src/InterestPayout.Worker/Messaging/Consumers/RecurringPayoutCommandConsumer.cs

```diff
@@ -216,6 +221,17 @@ public async Task Consume(ConsumeContext<RecurringPayoutCommand> context)
                         command
                     });
                     await unitOfWork.Commit();
+                    _cqrsEngine.PublishEvent(
```

Comment:
For `MeStatusCodes.Duplicate` it should do the same

Reply:
`+`